### PR TITLE
Ttahub 855/restore topics topic goals

### PR DESCRIPTION
**Pull request by GarrettEHill**
## Description of change
Restore and map old topics to new topics, reload TopicGoals table mapped to new values. Needed for the goal template release.

## How to test
Migration file does not fail any current tests. On a production, or sanitized copy of production, run the tools script using the commands as follows:

yarn docker:db:migrate
yarn docker:yarn:be restoreTopics:local

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-855

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: src/migrations/20220424000000-restore-topicgoals.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => queryInterface.sequelize.transaction(
+    async (transaction) => {
+      try {
+        const loggedUser = '0';
+        // const transactionId = '';
```
Comment (jasalisbury) [inline src/migrations/20220424000000-restore-topicgoals.js:6]: Should this be uncommented?

**Review by jasalisbury — COMMENTED** [review]
Does this need to run before or after the big migration? Or does it not matter?

File: package.json [hunk 2]
```diff
@@ -76,7 +79,9 @@
     "processData:local": "./node_modules/.bin/babel-node ./src/tools/processDataCLI.js",
     "changeReportStatus": "node ./build/server/tools/changeReportStatusCLI.js",
     "changeReportStatus:local": "./node_modules/.bin/babel-node ./src/tools/changeReportStatusCLI.js",
-    "makecolors": "node ./src/makecolors.js"
+    "makecolors": "node ./src/makecolors.js",
+    "restoreTopics": "node ./src/tools/restoreTopicsCLI.js",
```
Comment (kryswisnaskas) [inline package.json:83]:
```suggestion
    "restoreTopics": "node ./build/server/tools/restoreTopicsCLI.js",
```

File: src/tools/restoreTopics.js [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+      await sequelize.query(
+        `DO $$ BEGIN
+          PERFORM SETVAL('"TopicGoals_id_seq"', MAX(id)+1) FROM "TopicGoals";
+        END$$;`,
+        { transaction },
+      );
+    }));
+
+    // Disable logging while doing mass updates
```
Comment (kryswisnaskas) [inline src/tools/restoreTopics.js:137]:
```suggestion
    // Enable logging while doing structural updates
```

**Review by kryswisnaskas — COMMENTED** [review]
Looks good! 

There is only one change that will allow us to run the script in deployed environments. Other than that it works and it should be good to go. 👍

**Review by kryswisnaskas — APPROVED** [review]

**Pull request merged**